jquery.pjax.js

```diff
@@ -576,7 +576,6 @@ function stripParam(url, name) {
   return url
     .replace(new RegExp('[?&]' + name + '=[^&#]*'), '')
     .replace(/[?&]($|#)/, '\1')
-    .replace(/[?&]/, '?')
```

Comment:
I didn't give that a thought before, but you're right that the `_pjax` and `_` params will [likely always be appended](https://github.com/defunkt/jquery-pjax/blob/c4945b67869874e52cd19f4e1c900e3da5fdd2f4/jquery.pjax.js#L188-193). I just wanted my param stripping code to be agnostic of this fact.

We could definitely improve this function so it handles all cases well